src/webui/WebUI.jl

```diff
@@ -92,6 +115,11 @@ include("routes/callback.jl")
 include("routes/select.jl")
 include("routes/register.jl")
 include("routes/status.jl")
+include("routes/bitbucket.jl")
+
+auth_href(::Provider{GitLabAPI}) = "$(ROUTES[:AUTH])?provider=gitlab"
+auth_href(::Provider{GitHubAPI}) = "$(ROUTES[:AUTH])?provider=github"
+auth_href(::Provider{BitbucketAPI}) = "$(ROUTES[:BITBUCKET])/auth"
```

Comment:
Can't we use the same format `/auth/?provider=bitbucket`?

Reply:
No, Bitbucket puts its own query parameters in the URL -- it was probably a bad idea in the first place to assume it was safe to use query parameters

Reply:
I did try that in the beginning, btw -- if I remember correctly, bitbucket just replaced our parameters with its own